playwright/pages/epochs-dashboard.page.ts

```diff
@@ -24,9 +76,190 @@ export function epochsDashboardPage(page: Page) {
+    ).toEqual(currentEpoch?.end_time);
+
+    expect(
+      parseInt(<string>await totalBlockInEpochWidget.textContent()),
+      "Total block in finished epoch on widget to equal total block in epoch on Blockfrost "
+    ).toEqual(currentEpoch?.block_count || 0);
+
+    expect(
+      parseInt(<string>await txCountInEpochWidget.textContent()),
+      "Total transaction in finished epoch on widget to equal total block in epoch on Blockfrost "
+    ).toEqual(currentEpoch?.tx_count || 0);
+
+    expect(
+      +((await totalOutputInEpochWidget.textContent())?.replaceAll(",", "") || 0) * 10 ** 6,
+      "Total output in finished epoch on widget to equal total block in epoch on Blockfrost "
+    ).toEqual(+(currentEpoch?.output || 0));
+
+    await expect(blockTabWidget, "Block tab on epoch widget").toHaveText("Blocks");
+
+    await expect(viewDetailButtonWidget, "View details button on epoch widget").toHaveText("View details");
+  };
+
+  const checkEpochDetailPage = async ({ currentEpoch }: { currentEpoch?: BlockfrostEpochInformationDto }) => {
+    await expect(detailTitle, "Epoch detail title").toHaveText("Epoch Details");
+
+    expect(
+      moment((await startTimeEpochOverview.textContent())?.replace(",", "")).unix(),
+      "Start time on epoch detail to equal start time epoch Blockfrost"
+    ).toEqual(currentEpoch?.start_time);
+
+    expect(
+      moment((await endTimeEpochOverview.textContent())?.replace(",", "")).unix(),
+      "End time on epoch detail to equal end time epoch Blockfrost"
+    ).toEqual(currentEpoch?.end_time);
+
+    expect(
+      +((await totalOutputEpochOverview.textContent())?.replaceAll(",", "") || 0) * 10 ** 6,
+      "Total output in epoch detail to equal total block in epoch on Blockfrost "
+    ).toEqual(+(currentEpoch?.output || 0));
+
+    await expect(blockTableTitleEpochOverview, "Block table title").toHaveText("Blocks");
+
+    await expect(blockColumnBlockTable, "Check title on finished blocks table").toHaveText("Block");
+    await expect(blockIdColumnBlockTable, "Check title on finished blocks table").toHaveText("Block ID");
+    await expect(epochTitleColumnBlockTable, "Check title on finished blocks table").toHaveText("Epoch");
+    await expect(slotTitleColumnBlockTable, "Check title on finished blocks table").toHaveText("Slot");
+    await expect(slotNoTitleColumnBlockTable, "Check title on finished blocks table").toHaveText("Absolute Slot");
+    await expect(createdAtTitleColumnBlockTable, "Check title on finished blocks table").toHaveText("Created At");
+    await expect(txCountTitleColumnBlockTable, "Check title on finished blocks table").toHaveText("Transactions");
+    await expect(feesTitleColumnBlockTable, "Check title on finished blocks table").toHaveText("Fees");
+    await expect(outSumTitleColumnBlockTable, "Check title on finished blocks table").toHaveText("Output");
+  };
+
+  const checkBlockDetailPage = async () => {
+    expect(page.url().includes("/block")).toBe(true);
```

Comment:
When we check the redirect behavior to other pages in this case from epochs to block detail page will be nice make and additional but simple validation at least of the block number or block id depending of what the user selects.

So if the user selects the block number and he is redirected to the block detail page besides to check that the user was redirected to that page we should also add a validation about the user was redirected to the selected block detail page by comparing the selected block number with the block number that is displayed in the block detail page.

Besides like this is a validation over the block detail page this and the respective elements/locators should be in the block-detail.page.ts